### PR TITLE
Setup CI

### LINKED ISSUE #11: setup project CI

### PR DESCRIPTION
**Pull request by hoanhan101**
[Fix #11] So I tried to mirror what has been done in [blackbox](https://github.com/vapor-ware/blackbox) and this is what I came up with.

### CODE REVIEW

**Review by lazypower — APPROVED** [review]
overall this looks good to me. A note for possible improvement of the pipeline - marco's working on getting goreleaser deployed to jenkins, I believe its already installed on the host. You may wish to give that a go. but I'm also totally good with adopting this build pipeline that I've seen in synse-emulator. we can circle back for goreleaser at a later date.

:+1:

**Comment by hoanhan101** [comment]
@lazypower Thank you sir! I am opening an issue for that.

File: .jenkins [hunk 1]
```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/env groovy
+
+pipeline {
+
+  agent any
+
+  environment {
+    PROJ_PATH = '/go/src/github.com/vapor-ware/synse-client-go'
+    IMAGE_NAME = 'vaporio/synse-client-go'
```
Comment (edaniszewski) [inline .jenkins]: we can probably remove this env var, since we aren't building an image for this

**Review by edaniszewski — APPROVED** [review]
one super minor note, but otherwise LGTM!

**Pull request merged**